frontend/src/app/layout.tsx

```diff
@@ -28,8 +29,9 @@ export default function RootLayout({
       <body
         className={`${geistSans.variable} ${geistMono.variable} antialiased`}
       >
+        <SideNavBar />
         {children}
-        <Footer />
+      <Footer />
```

Comment:
Nit: indentation is wrong. We should be ideally using a formatter that we can keep consistent across the team.

Reply:
Fixed it.